embc-app/ClientApp/src/app/volunteer/pages/referral-maker/referral-maker.component.html

```diff
@@ -10,59 +10,84 @@ <h1>Identify Supports</h1>
 
     <section class="clearfix">
       <!-- confirm task start time -->
+
       <div class="feature-block card d-block border-0">
-        <div *ngIf="registration?.incidentTask['startDate']">
-          <label>Please confirm timing for commencement of supports:</label>
-          <div>Incident start date &amp; time: {{registration.incidentTask['startDate'] | dateTimeFormatPipe}}</div>
+        <div class="card-body">
+          <div *ngIf="registration?.incidentTask['startDate']">
+            <!-- <div>{{registration.incidentTask['startDate'] | date:'yyyy/MM/dd \'at\' HH:mm:ss a'}}</div> -->
+            <div>Incident start date &amp; time: {{registration.incidentTask['startDate'] | dateTimeFormatPipe}}</div>
+
+            <div class="row">
+              <div class="col">
+                <span class="align-bottom">
+                  <span class="h3">Support start date &amp; time: {{defaultDate | dateTimeFormatPipe}}</span>
+                </span>
+              </div>
+              <div class="col-2">
+                <span class="align-bottom">
+                  <a href="javascript:void(0)">
+                    <app-font-awesome-icon icon="times"></app-font-awesome-icon>
+                    <span class="ml-2" (click)="showDefaultDatePicker=!showDefaultDatePicker">Override start time</span>
+                  </a>
+                </span>
+              </div>
+            </div>
+          </div>
+          <div *ngIf="showDefaultDatePicker">
+            <strong>Valid From</strong>
+            <app-date-time-picker [(value)]="defaultDate"></app-date-time-picker>
+            <button class="btn btn-primary mt-2" (click)="showReferralMakers=!showReferralMakers">NEXT</button>
```

Comment:
NEXT button needs to always be visible. My previous comment was about moving it below the override inputs so that the user flow is "top to bottom" of the page.

Also, currently the NEXT button toggles the supports checkboxes (which it shouldn't)... maybe a simple state machine needs to be implemented to manage what step the user is on?